### PR TITLE
Bug 1181198

### PR DESCRIPTION
**Pull request by tannishk**
Refactor all javascript to use 4 spaces for indentation

### CODE REVIEW

**Comment by Nolski** [comment]
Looking good! However it looks like [airmozilla/popcorn/static/popcorn/js/editor.js](https://github.com/mozilla/airmozilla/blob/master/airmozilla/popcorn/static/popcorn/js/editor.js) still uses 2 spaces for indentation.

**Comment by peterbe** [comment]
Hi @tannishk Thanks for taking this one!
It seems one of the files have changed since you started working on this. Do you know how to rebase your local branch against the latest upstream master?
If that sounds like a greek to you, I'm happy to help you walk through it. Even better, jump onto IRC.mozilla.org and join the `#airmozilla-dev` room and say hi to `peterbe` or `nolski`.

**Comment by tannishk** [comment]
@peterbe @Nolski Check now

File: airmozilla/popcorn/static/popcorn/js/editor.js [hunk 1]
```diff
@@ -1,41 +1,37 @@
...
+      .done(function (response) {
+          var message = "Your edit has been saved and sent it for transcoding. "
+          + "Once it completes, which can take several minutes, we'll automatically"
+          + " update the event to use your latest video edits.";
+          alert(message);
+      })
+      .fail(function () {
+          console.warn('Unable to save edit :(');
+          console.error.apply(console, arguments);
+       });
```
Comment (peterbe) [inline airmozilla/popcorn/static/popcorn/js/editor.js]: This file got worse :)
 It *was* 4-spaces everywhere now there's some lines that are 2-spaces. E.g. between line 2 and 3.

**Comment by Nolski** [comment]
It looks like this is a duplicate of the work submitted in #437

**Comment by peterbe** [comment]
@tannishk To get this unstuck, I suggest you check out the `editor.js` in master. I.e. void your work on that file. Then your PR will be fixing 2 files.

**Comment by tannishk** [comment]
@peterbe editor.js back to 4 space indentation

**Comment by peterbe** [comment]
`editor.js` is still full of changes. Most of them still wrong.

**Comment by tannishk** [comment]
@peterbe

**Comment by peterbe** [comment]
@tannishk Hey. Sorry for late reply. I must have missed your last comment. 

No, the file still uses 2 spaces indentation.

**Pull request closed without merging**